tests/jvm/src/test/scala/scala/meta/tests/cli/CliSuite.scala

```diff
@@ -20,7 +20,7 @@ class CliSuite extends FunSuite with DiffAssertions {
     }
   """.getBytes(UTF_8))
   val target = Files.createTempDirectory("target_")
-  val helloWorldSemanticdb = target.resolve("META-INF/semanticdb/HelloWorld.semanticdb")
+  val helloWorldSemanticdb = target.resolve("META-INF/semanticdb/HelloWorld.scala.semanticdb")
```

Comment:
@olafurpg This is potentially controversial, but we don't provide any [compatibility guarantees](https://github.com/scalameta/scalameta/blob/master/COMPATIBILITY.md) for the file layout.

Reply:
Review of our tools:
  * `semanticdb-scala` [already does the same](https://github.com/scalameta/semanticdb-sbt/blob/master/sbthost/nsc/src/main/scala/scala/meta/internal/sbthost/ConfigOps.scala#L10).
  * `Scalafix` [uses Database.load](https://github.com/scalacenter/scalafix/blob/master/scalafix-cli/src/main/scala/scalafix/cli/CliRunner.scala#L345), which in turn [calls v.Database.toSchema](https://github.com/scalameta/scalameta/blob/master/langmeta/langmeta/shared/src/main/scala/org/langmeta/semanticdb/Database.scala#L47), which [doesn't care about on-disk filenames](https://github.com/scalameta/scalameta/blob/master/langmeta/langmeta/shared/src/main/scala/org/langmeta/semanticdb/internal/vfs/Database.scala#L22) as long as [they end with "semanticdb"](https://github.com/scalameta/scalameta/blob/master/langmeta/langmeta/shared/src/main/scala/org/langmeta/semanticdb/internal/vfs/SemanticdbPaths.scala#L13).
  * Metadoc already does the same (multiple links don't fit here).
  * Metals doesn't care about filenames at all.

Reply:
Just do it. I don't rely on the filenames because the URI is embedded in the proto message